torch/utils/cpp_extension.py

```diff
@@ -1865,7 +1865,8 @@ def load_inline(name,
                 is_python_module=True,
                 with_pytorch_error_handling=True,
                 keep_intermediates=True,
-                use_pch=False):
+                use_pch=False,
+                no_implicit_headers=False):
```

Comment:
nit: kwarg only might be good for these haha
Also there are some people that argue that positive flags are best as the readability of `if not no_implicit_headers:` is not the best :D